recipes/xsel/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "xsel" %}
+{% set version = "1.2.1" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh

```

Comment:
I don't see this used anywhere in `build.sh`

Reply:
Done.